### ISSUE TITLE
Error: 'col has to be greater than 1'

### ISSUE THREAD

**Issue by derN3rd** [jroehl/gsheet.action#572]
**Describe the bug**

Hey there, I'm experiencing an error while using the action in our deployment script where I try to use the `appendData` function to write into a gsheet that we use as a changelog.

When I add `minCol: 2` to the args it works, but it's starting from the second columns only, which is incorrect (or at least not what I want, as I need to fill in column 1 as well).

Not exactly sure how and why he fails, or what the issue is with the col:1 config.

**To Reproduce**
1. Add the action as config shown below
2. Run the action
3. Error is happening

**Expected behavior**
Should not fail with this error.

**Screenshots**

![Screenshot 2023-07-05 at 17 31 03](https://github.com/jroehl/gsheet.action/assets/7534045/51376b2d-216c-4648-839e-f3be4b5cb87f)

**Desktop (please complete the following information):**
 - Version: 2.0.0

**Additional context**
Action config:
```yaml
name: Deploy Service

jobs:
  
  [...]
  
  deploy-api:
    runs-on: ubuntu-latest
    timeout-minutes: 35
    needs: build-and-push-docker-image
    environment:
      name: ${{ github.event.inputs.deployment_target }}
    name: Deploy API
    steps:
     [...]
      - name: "Add to production changelog"
        uses: jroehl/gsheet.action@v2.0.0
        # only run if if deployment target name includes "production" or staging
        if: contains(inputs.deployment_target, 'production') || contains(inputs.deployment_target, 'staging')
        with:
          spreadsheetId: xxxxxxx
          commands: |
            [
              { "command": "appendData", "args": { "spreadsheetId": "xxxxxxx", "worksheetTitle":"Main", "data": [["${{ steps.current_iso_time.outputs.json_date }}", "${{ steps.current_iso_time.outputs.json_time }}", "${{ inputs.deployment_target }}", "", "${{ steps.changelog.outputs.changelog-gsheet }}", "${{ steps.version.outputs.label }}"]] }}
            ]
        env:
          GSHEET_CLIENT_EMAIL: scripts@xxx-app.iam.gserviceaccount.com
          GSHEET_PRIVATE_KEY: ${{ secrets.GOOGLE_SHEET_PRIVATE_KEY }}
```

**Comment by derN3rd** [comment]
@jroehl do you have any idea how can debug this further to find the source of the error?

Didn't find any of the error message inside this action or your google-sheet-cli, so it seems to be an error returned by Google Sheets API.

Is there a way to enable debug output for the action and the underlying google-sheet-cli, to easier debug this?

Thanks in advance

**Comment by derN3rd** [comment]
Seems like the default `minCol:1` is not set as described in the readme. Adding it manually to the command array fixes the issue.

The error message is kinda confusing, as the check does `col < 1` while the message states it must be greater than one, while its `1 or greater`.

Hope this helps someone

**Comment by nestorlafon** [comment]
This is closed but I still see the same issue in 2.1.0. Thanks for sharing how you solved it @derN3rd!
